docs/source/projects.rst

```diff
@@ -87,6 +88,28 @@ Conda environment
   file in the root of the project directory or by including a ``conda_env`` entry in your
   ``MLproject`` file. For details, see the :ref:`project-directories` and :ref:`mlproject-specify-environment` sections.
 
+  The ``mlflow run`` command supports running a Conda environment project as a Python environment project.
```

Comment:
Do we want to say "local Python environment" or leave it as "Python environment"?
I've thought of it as a hierarchy of 
Python Environment is a generic term implying that Python is available and installed/activated somewhere. 
That 'somewhere' could be:
Local, Virtualenv, Conda, or Docker, et al.

Reply:
> Python Environment is a generic term implying that Python is available and installed/activated somewhere.


Agree, what do you think about `Virtualenv project`?

Reply:
@dbczumar wdyt?

Reply:
One concern is matching to the `python_env` field.

- conda -> conda_env
- docker -> docker_env
- virtualenv -> python_env

Reply:
`Virtualenv project` makes sense. `virtualenv -> python_env` is also just fine.